docs/index.md

```diff
@@ -2,57 +2,69 @@
 
 ## Overview
 
-Open Traffic Generator (OTG) APIs and Data Models are APIs to test Layer 2 & 3 routers and switches. They include support for testing the forwarding plane as well as testing the control plane.  They are:
+Open Traffic Generator (OTG) APIs and data models are northbound API specifications for modern Traffic Generators and Protocol Emulators designed to test Layer 2-7 network devices. They include support for testing  forwarding as well as control planes. OTG APIs are:
 
 * Open
-* Vendor Neutral
-* Intent Based
+* Vendor-neutral
+* Intent-based
 * Declarative
 
-The specification is available on GitHub [here](https://github.com/open-traffic-generator/models/blob/master/artifacts/openapi.yaml) (viewer friendly version at reDocly [here](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml)).  Specifications support both REST or gRPC interfaces.
+The formal [model specification](https://github.com/open-traffic-generator/models/blob/master/artifacts/openapi.yaml) can be found on GitHub under [Open Traffic Generator](https://github.com/open-traffic-generator) organization. To explore the model, a viewer friendly [**ReDoc rendering**](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/open-traffic-generator/models/master/artifacts/openapi.yaml) is available as well. The OTG APIs support both REST and gRPC interfaces.
 
 ## Features
 
-OTG is an actively developing specification, with contributions coming from real [use cases](/examples/#use-cases). On high level, the model allows to express the following building blocks of a traffic generator configuration:
- 
-* *Test Ports* with:
-	- LLDP
-	- LAG
-	- LACP
-* *Traffic Flows* associated with:
-	- Test Ports
-	- Emulated Devices
-* *Emulated Devices* with:
-	- BGP
-	- IS-IS
+OTG is an actively developing specification, with contributions from real [use cases](/examples/#use-cases). The model allows to express the following building blocks of a traffic generator configuration:
+<!-- TODO add links from bold items to paragraphs in Model section -->
+* **Test Ports** with Layer 1&2 capabilities, including:
+	- LLDP, LAG, LACP
+* Emulated **OTG Devices** with Layer 2&3 features:
+	- IPv4, IPv6 interfaces
+	- ARP, IPv6 ND
+	- BGP, IS-IS routing protocols
+* **Traffic Flows** 
+    - associated with either Test Ports, or OTG Devices
+	- expressing L2-4 properties like Ethernet, IPv4/IPv6, TCP/UDP
+	- stateless or stateful capabilities for transport protocols
+	- with implementation-specific application payload
+* Run-time **Metrics** and traffic **Capture** capabilities
+
+![Example OTG Diagram](images/otg-example-diagram.svg)
+<sub>Fig. 1. Sample diagram of the OTG configuration with emulated BGP routers, traffic flows and a Device Under Test</sub>
 
 
 ## Implementations
```

Comment:
Do you want to repeat the contents from implementations.md here?

Reply:
It is the other way. At the moment the content in `implementations.md` is a stub that I copied from here.